common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerOperationExecutor.java

```diff
@@ -232,6 +234,9 @@ private <T> T performStrategy(@NonNull final IIpcStrategy strategy,
             span.setAttribute(AttributeName.ipc_strategy.name(), strategy.getType().name());
             operation.performPrerequisites(strategy);
             final BrokerOperationBundle brokerOperationBundle = operation.getBundle();
+            if (brokerOperationBundle.getBundle() != null) {
```

Comment:
if bundle is null, create an empty bundle with the flag?

Reply:
bundle is final so I can move it inside brokerOperationBundle